package.json

```diff
@@ -22,9 +22,14 @@
   },
   "readmeFilename": "README.md",
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
```

Comment:
I think we should add these dependencies in solstice assets. I think we will need them outside of hugo.

Reply:
moved to assets